Sources/SwiftCurrent/Models/Workflow.swift

```diff
@@ -49,7 +49,7 @@ public final class Workflow<F: FlowRepresentable>: LinkedList<_WorkflowItem> {
      Launches the `Workflow`.
      - Parameter orchestrationResponder: the `OrchestrationResponder` to notify when the `Workflow` proceeds or backs up.
      - Parameter launchStyle: the launch style to use.
-     - Parameter onFinish: the closure to call when the last element in the workflow proceeds; called with the `AnyWorkflow.PassedArgs` the workflow finished with.
+     - Parameter onFinish: the closure to call when the last element in the workflow proceeds; called with the `AnyWorkflow.PassedArgs` of the last node in the workflow.
```

Comment:
This is no longer a true statement. I doesn't end with a preposition, but it's false.

Reply:
Reverted all instances of this change